src/components/fx/hover.js

```diff
@@ -772,18 +772,19 @@ function createHoverText(hoverData, opts, gd) {
         var commonBgColor = commonLabelOpts.bgcolor || Color.defaultLine;
         var commonStroke = commonLabelOpts.bordercolor || Color.contrast(commonBgColor);
         var contrastColor = Color.contrast(commonBgColor);
+        var commonLabelFont = {
+            family: commonLabelOpts.font.family || fontFamily,
+            size: commonLabelOpts.font.size || fontSize,
+            color: commonLabelOpts.font.color || contrastColor
```

Comment:
Do we always have `commonLabelOpts.font` in the options?
If not, we may use 
```js
(commonLabelOpts.font || {}).etc
```.

Reply:
Yep! If not many things would have been broken for a long time.